server/storage/errors.go

```diff
@@ -40,3 +40,13 @@ type ErrNoKey struct {
 func (err ErrNoKey) Error() string {
 	return fmt.Sprintf("Error, no timestamp key found for %s", err.gun)
 }
+
+// ErrBadChangeID indicates the change ID provided by the user is not
+// valid for some reason, i.e. it is out of bounds
+type ErrBadChangeID struct {
+	id string
+}
+
+func (err ErrBadChangeID) Error() string {
```

Comment:
Huh, not sure how I ended up not using this... I know I did at some point. I need to make one more tiny tweak to tell the JSON encoding to return the change ID as a string so I'll take this out at the same time.